troubleshooting.md

```diff
@@ -18,3 +18,17 @@ Most likely not an actual error. Your server does not support IPv6. Can be ignor
 
 When running the CoD4x \(linux\) server inside a docker container on a windows machine the main and zone folders should not be mounted directly, but be mounted on its parent path. Otherwise the server will fail to load the iwds for some reason.
 
+#### How do I take a screenshot of a player? 
+
+In your server console, type `status`. This will display a list of users that are currently connected to your server. Take the number of the user off of that list, or type their user name into this command.... i.e `getss 3`. Once the command is issued to the server it will save the screenshot of the user under a new folder called `screenshots` in your main server directory. 
```

Comment:
This is not entirely accurate. The `screenshots` directory is created wherever `fs_homepath` points to, which I believe defaults to `${HOME}/.callofduty4`. You can also specify the filename explicitly to the `getss` command. The syntax of the command is `getss <player|all> [filename]`.